### ISSUE TITLE
Galleria Component Accessibility with keyboard

### ISSUE THREAD

**Issue by Maikel-Nait** [primefaces/primefaces#910]
It should be possible to navigate through the Galleria images using the keyboard ( at least with the arrow keys ). It would be a nice addition to PF 5.4, as it is mainly focused on accessibility.

**Comment by Maikel-Nait** [comment]
BTW, so far, the HotKey PrimeFaces component allows using cursor keys with the galleria component. Just thinking if it could be the default behaviour ...

**Comment by melloware** [comment]
- Added LEFT and RIGHT keyboard events if the galleria has focus
- Added tabIndex=0 so the galleria can receive focus and keyboard events
- Add touch swipe events for mobile browsers to swipe back and forth.

**Comment by Maikel-Nait** [comment]
Better late than never :)
Awesome !
